packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -172,6 +187,7 @@ ContractDetails.propTypes = {
     duration_unit: PropTypes.string,
     exit_spot: PropTypes.string,
     is_vanilla: PropTypes.bool,
+    symbol: PropTypes.string,
```

Comment:
Where symbol is used?

Reply:
This can be removed too, I removed symbol from this file